packages/fetchai/connections/soef/connection.py

```diff
@@ -1286,21 +1292,29 @@ def __init__(self, **kwargs: Any) -> None:
                 self.DEFAULT_CONNECTION_CHECK_MAX_RETRIES,
             ),
         )
+        is_https = cast(bool, self.configuration.config.get("is_https"))
```

Comment:
```suggestion
        is_https = cast(bool, self.configuration.config.get("is_https"), True)
```
Makes sense to just set a default for backwards compatibility in tests etc.